charts/timber-fluentd/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.gcpServiceAccount.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ .Release.Name }}-fluentd-secret
```

Comment:
Can we consistently use {{ include "timber-fluentd.fullname" . }} instead of {{ .Release.Name }} ?